yarn.lock

```diff
@@ -21676,7 +21676,7 @@ js-yaml@4.1.0, js-yaml@^4.1.0:
   dependencies:
     argparse "^2.0.1"
 
-js-yaml@^3.13.1, js-yaml@^3.14.1:
+js-yaml@^3.13.1:
```

Comment:
This is coming from `@istanbuljs/load-nyc-config`, we can also set a resolution, though seems negligible for now
```json
"**/@istanbuljs/load-nyc-config/**/js-yaml": "4.1.0"
```

Reply:
I agree. As this is a transitive development dependency, I think it is ok to leave this for now.